catalogs/data/bedrock/helm/bedrock.yaml.liquid

```diff
@@ -0,0 +1,21 @@
+{% raw %}
+{% assign imports_bedrock_key =  'bedrock-' | append: configuration.name | append: '-' |  append: configuration.cluster %}
+
+config:
+  provider: bedrock
+
+args:
+  - --provider=bedrock
+
+env:
+  - name: AWS_REGION
```

Comment:
EKS IRSA should set this by default, so there shouldn't be a need for us to wire it into helm and have it be an input